File: quick-start/node/outbound-call.ts
import { config } from "dotenv";
import { PhonicClient } from "phonic";

config({ path: ".env.local" });

const client = new PhonicClient({
  apiKey: process.env.PHONIC_API_KEY,
  baseUrl: process.env.PHONIC_API_BASE_URL,
});

async function outboundCall() {
  await client.conversations.outboundCall({
    to_phone_number: "YOUR_PHONE_NUMBER", // e.g. +19189391262
    config: {
      agent: "my-first-agent",
      welcome_message: "Hello, how can I help you?",
    },
  });
}

await outboundCall();

File: 01-quick-start/python/outbound_call.py
import os

from dotenv import load_dotenv
from phonic import Phonic

load_dotenv(".env.local")

client = Phonic(
    api_key=os.getenv("PHONIC_API_KEY"),
)


def outbound_call():
    client.conversations.outbound_call(
        to_phone_number=os.getenv("CUSTOMER_PHONE_NUMBER"),
        config={
            "agent": "my-first-agent",
            "welcome_message": "Hello, how can I help you?",
        },
    )


if __name__ == "__main__":
    outbound_call()
